CONTRIBUTING.md

```diff
@@ -18,10 +18,10 @@ Here are some easy things you can always do:
 
 ## Translations
 
-It would be great if you can contribute your translations! You can either translate the JSON files directly or use [this online translator service](https://lusito.github.io/web-ext-translator/?gh=https://github.com/rugk/mastodon-simplified-federation).
+It would be great if you can contribute your translations! You can either translate the JSON files directly or use [this online translator service](https://lusito.github.io/web-ext-translator/?gh=https://github.com/rugk/mastodon-simplified-federation/tree/main).
 
 **Manually:** To translate it manually, go to [`src/_locales/en`](src/_locales/en) and copy the English (or any other existing language) `messages.json` file. (You could also use another source language if you want, but usually English is the best.) Create a new dir at `src/_locales` with the abbreviation of the language you want to translate.  
-**Web-ext-translator:** Go to [this page](https://lusito.github.io/web-ext-translator/?gh=https://github.com/rugk/mastodon-simplified-federation/) and translate it online. Download the result by clicking on "Export to ZIP" at the bottom.
+**Web-ext-translator:** Go to [this page](https://lusito.github.io/web-ext-translator/?gh=https://github.com/rugk/mastodon-simplified-federation/tree/main) and translate it online. Download the result by clicking on "Export to ZIP" at the bottom.
```

Comment:
Ah thanks a lot, never noticed the service implies `master` as being the default?

Would you maybe mind to raise an issue in https://github.com/Lusito/web-ext-translator/issues/? Maybe it could check these two branches at least, so this explicit link is not necessary.
After all `main` is the default nowadays and many projects/templates use it by default.

Reply:
Okay did it here now: https://github.com/Lusito/web-ext-translator/issues/82